src/app/app.module.ts

```diff
@@ -10,9 +10,12 @@ import {
 import { CommonModule } from '@angular/common';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { MenuModule } from './components/menu/menu.module';
+import { FooterComponent } from './components/footer/footer.component';
+import {MatCardModule} from "@angular/material/card";
+import {HomeComponent} from "./components/home/home.component";
```

Comment:
Sometimes there is empty space after and before `{` / `}` - this should be unified. Same for using sometimes `"` and `'` -> I would always use single quotes.